adminSiteClient/EditorDebugTab.tsx

```diff
@@ -186,6 +194,143 @@ class EditorDebugTabForChart extends React.Component<{
     }
 }
 
+@observer
+class EditorDebugTabForChartView extends React.Component<{
+    editor: ChartViewEditor
+}> {
+    @action.bound copyYamlToClipboard() {
+        // Avoid modifying the original JSON object
+        // Due to mobx memoizing computed values, the JSON can be mutated.
+        const patchConfig = {
+            ...this.props.editor.patchConfig,
+        }
+        delete patchConfig.id
+        delete patchConfig.dimensions
+        delete patchConfig.version
+        delete patchConfig.isPublished
+        const chartConfigAsYaml = YAML.stringify(patchConfig)
+        // Use the Clipboard API to copy the config into the users clipboard
+        void copyToClipboard(chartConfigAsYaml)
+        notification["success"]({
+            message: "Copied YAML to clipboard",
+            description: "You can now paste this into the ETL",
+            placement: "bottomRight",
+            closeIcon: <></>,
+        })
+    }
+
+    @observable diffModalOpen = false
+
+    @action.bound onModalClose() {
+        this.diffModalOpen = false
+    }
+
+    @computed get diffModal() {
+        return (
+            <Modal
+                open={this.diffModalOpen}
+                centered
+                width="80vw"
+                onOk={this.onModalClose}
+                onCancel={this.onModalClose}
+                cancelButtonProps={{ style: { display: "none" } }}
+            >
+                <div style={{ maxHeight: "50vh", overflowY: "auto" }}>
+                    <ReactDiffViewer
+                        newValue={stringify(
+                            this.props.editor.fullConfig,
+                            null,
+                            2
+                        )}
+                        oldValue={stringify(
+                            this.props.editor.parentConfig,
+                            null,
+                            2
+                        )}
+                        leftTitle={"a"}
+                        rightTitle={"b"}
+                        compareMethod={DiffMethod.WORDS_WITH_SPACE}
+                        styles={{
+                            contentText: {
+                                wordBreak: "break-word",
+                            },
+                        }}
+                        extraLinesSurroundingDiff={2}
+                    />
+                </div>
+            </Modal>
+        )
+    }
+
+    render() {
+        const { patchConfig, parentConfig, fullConfig, parentChartId } =
+            this.props.editor
+
+        const parentChartLink = (
+            <a
+                href={`/admin/charts/${parentChartId}/edit`}
+                target="_blank"
+                rel="noopener"
+            >
+                {parentConfig?.title ?? "(missing title)"}
+            </a>
+        )
+
+        return (
+            <div>
+                <Section name="Config">
+                    <textarea
```

Comment:
I get a squiggly line here that this needs a label. It's debug functionality but maybe worth it for a cleaner editor experience :)

Reply:
Must be some kind of editor (accessibility?) plugin that you have and I don't?
Anyhow, I tried quickly adding one and then the layout broke, and I couldn't be arsed 😀